test/utils.ts

```diff
@@ -1,5 +1,7 @@
 const compression = require('compression');
 const webserver = require('../lib/webserver');
+const express = require('express')
```

Comment:
comment: modified this to accommodate the fact that our requests to the git client have a `snykgit` prefix. My only thought was that the tests might be more generic (instead of using `snykgit` we could have `special-path` for example)